drivers/sensor/tmag5273/Kconfig

```diff
@@ -0,0 +1,367 @@
+# TMAG5273 Low-Power Linear 3D Hall-Effect Sensor configuration options
+
+# Copyright (c) 2022 deveritec GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig TMAG5273
+	bool "TMAG5273 3D Hall-Effect Sensor"
+	depends on I2C
+	help
+		Enable driver for TMAG5273 I2C-based 3d Hall-Effect sensor.
+
+if TMAG5273
+
+comment "measurement"
+
+config TMAG5273_MAGNETIC_CH_X_EN
+	bool "Measure x-axis"
+	default y
+	select TMAG5273_MAGNETIC_EN
+
+config TMAG5273_MAGNETIC_CH_Y_EN
+	bool "Measure y-axis"
+	default y
+	select TMAG5273_MAGNETIC_EN
+
+config TMAG5273_MAGNETIC_CH_Z_EN
+	bool "Measure z-axis"
+	default y
+	select TMAG5273_MAGNETIC_EN
+
+config TMAG5273_MAGNETIC_EN
+	bool
+
+config TMAG5273_TEMP_MEAS_EN
+	bool "Enable temperature measurement"
+	default n
```

Comment:
Remove this line. It's redundant because bool configs are default n unless defined otherwise